tests/test_script_train.py

```diff
@@ -4,5 +4,5 @@
 
 
 def test_train():
-  train_ex.run(named_configs=['cartpole', 'gail', 'debug'])
+  train_ex.run(named_configs=['cartpole', 'gail', 'debug', 'test_data'])
```

Comment:
Consider using `config_updates` instead of this additional named config.

Also shouldn't `debug` be called `fast` now?